### PR TITLE
Form reformatting

### PR DESCRIPTION
**Pull request by pkakelas**
I didn't reformat id tag to work with this way, mainly because it is the main styling tag and we will use it very frequently

### CODE REVIEW

File: helpers/html.php [hunk 1]
```diff
@@ -73,17 +74,21 @@ public function createInput( $type = 'text', $name = '', $id = '', $value = '',
...
         }
 
-        public function createSubmit( $value ) {
-            $this->createInput( 'submit', '', '', $value );
+        public function createSubmit( $value, $attributes = '' ) {
+            $this->createInput( 'submit', '', '', $value, $attributes );
         }
 
-        public function createSelect( $name = '', $option_array, $selected = '', $id = '' ) {
+        public function createSelect( $name = '', $option_array, $selected = '', $id = '', $attributes = '' ) {
```
Comment (dionyziz) [inline helpers/html.php]: Optional arguments must follow obligatory arguments.

File: helpers/html.php [hunk 2]
```diff
@@ -95,6 +100,13 @@ public function createSelect( $name = '', $option_array, $selected = '', $id = '
                         echo htmlspecialchars( $id );
                     ?>" <?php
                 }
+                if ( !empty( $attributes ) ) {
+                    foreach ( $attributes as $key => $value ) {
+                        echo $key ?>="<?php
```
Comment (dionyziz) [inline helpers/html.php]: New line before `?>`

File: helpers/html.php [hunk 3]
```diff
@@ -162,6 +174,13 @@ public function output( $callable = false ) {
                         echo htmlspecialchars( $this->id );
                     ?>" <?php
                 }
+                if ( isset( $this->attributes ) ) {
+                    foreach( $this->attributes as $key => $value ) {
```
Comment (dionyziz) [inline helpers/html.php:180]: space before `(`

File: helpers/html.php [hunk 4]
```diff
@@ -162,6 +174,13 @@ public function output( $callable = false ) {
...
+                        echo $key ?>="<?php
```
Comment (dionyziz) [inline helpers/html.php]: New line before `?>`

File: helpers/html.php (hunk 2 above)
Comment (vitsalis) [inline helpers/html.php]: you forgot a `;`

File: helpers/html.php [hunk 5]
```diff
@@ -73,17 +74,21 @@ public function createInput( $type = 'text', $name = '', $id = '', $value = '',
                         echo htmlspecialchars( $value );
                     ?>" <?php
                 }
-                if ( $type == 'checkbox' && $checked ) {
-                    ?>checked="yes"<?php
+                if ( !empty( $attributes ) ) {
+                    foreach( $attributes as $key => $value ) {
+                        echo $key ?>="<?php
```
Comment (vitsalis) [inline helpers/html.php]: you forgot a `;`

Comment (vitsalis) [inline helpers/html.php]: new line before `?>`

**Pull request merged**